### PR TITLE
Remove Code as a selectable filetype when adding a file

### PR DESCRIPTION
**Pull request by rebkwok**

### CODE REVIEW

File: airlock/forms.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class AddFileForm(forms.Form):
         choices=[
             (i.name, i.name.title())
             for i in RequestFileType
-            if i != RequestFileType.WITHDRAWN
+            if i not in [RequestFileType.WITHDRAWN, RequestFileType.CODE]
```
Comment (madwort) [inline airlock/forms.py:19]: should we just allow-list?

```suggestion
            if i in [RequestFileType.OUTPUT, RequestFileType.SUPPORTING]
```

File: tests/unit/test_forms.py [hunk 2]
```diff
@@ -12,6 +12,14 @@ def test_add_file_form_no_release_request():
     assert form.fields["filegroup"].choices == [("default", "default")]
 
 
+def test_add_file_form_file_type_choices():
+    form = AddFileForm(release_request=None)
+    assert form.fields["filetype"].choices == [
```
Comment (madwort) [inline tests/unit/test_forms.py:17]: thanks for adding a test!

File: airlock/forms.py (hunk 1 above)
Comment (rebkwok) [reply airlock/forms.py:19]: Maybe? I thought about it, wasn't sure if there could ever be another type in future. There's a test now at least, if we did this again

Comment (madwort) [reply airlock/forms.py:19]: 🤷 I wouldn't have expected CODE to be added tbh, but not really fussed.

**Review by madwort — APPROVED** [review]

**Pull request merged**